docs/sources/dashboards/previews.md

```diff
@@ -0,0 +1,112 @@
+
+1. Install the Image Renderer plugin or set up a remote rendering service. `3.4.0` is the minimum version of Image Renderer required for the dashboard preview feature. Refer to [Image rendering]({{< relref "../image-rendering/" >}}) for more information.
+2. Open the [config file]({{< relref "../administration/configuration.md#configuration-file-location" >}}) and enable the `dashboardPreviews` [feature toggle]({{< relref "../administration/configuration.md#feature_toggles" >}}).
+
+```
+[feature_toggles]
+# enable features, separated by spaces
+enable = dashboardPreviews
+```
+
+3. Save your changes. Grafana should reload automatically; we recommend restarting the Grafana server in case of any issues.
+
+Visit the dashboard search page to verify that the setup was successful. You should be able to see dashboard preview placeholders for all your existing dashboards after clicking on the _Show previews_ toggle at the top of page.
+
+{{< figure src="/static/img/docs/dashboards/previews-successful-setup.jpg"
+max-width="950px" animated-gif="/static/img/docs/dashboards/previews-successful-setup.gif" >}}
+
+Please check [Grafana server logs]({{< relref "../administration/configuration.md#log" >}}) in case you see any warnings after clicking on the toggle - errors logs produced by `dashboard_service` should provide more context.
+
+{{< figure src="/static/img/docs/dashboards/previews-unsuccessful-setup.png" max-width="950px" >}}
+
+## About the dashboard previews crawler
+
+The dashboard previews crawler is a background process that:
+
+1. [Prepares a list of dashboards to visit](#preparing-the-dashboard-list)
+2. [Visits and takes a screenshot of each dashboard](#rendering-previews)
+3. [Saves the screenshots in persistent storage](#saving-previews)
+
+The crawler can be configured via the main config file:
+
+```ini
+[dashboard_previews.crawler]
+# Number of dashboards rendered in parallel. Default is 6.
+thread_count =
+
+# Timeout passed down to the Image Renderer plugin. It is used in two separate places within a single rendering request:
+# First during the initial navigation to the dashboard and then when waiting for all the panels to load. Default is 20s.
+# This setting should be expressed as a duration. Examples: 10s (seconds), 1m (minutes).
+rendering_timeout =
+
+# Maximum duration of a single crawl. Default is 1h.
+# This setting should be expressed as a duration. Examples: 10s (seconds), 1m (minutes).
+max_crawl_duration =
+
+# Minimum interval between two subsequent scheduler runs. Default is 12h.
+# This setting should be expressed as a duration. Examples: 10s (seconds), 1m (minutes).
+scheduler_interval =
+```
+
+### Preparing the dashboard list
+
+During the initial crawler run, the list will contain all dashboards across all organizations.
+The list will have fewer elements during subsequent runs - it will contain only dashboards which have changed since taking the last preview and those which are brand-new.
```

Comment:
```suggestion
During the initial crawler run, the list contains all dashboards across all organizations.
During subsequent runs, the list will have fewer elements. For example, it will only contain dashboards that have changed since taking the last preview and those which are brand-new.
```

Reply:
applied the first sentence and second. Turned the last sentence into bulletpoints - I am not sure about the exact wording